content/text-to-speech.md

```diff
@@ -35,10 +35,9 @@ If you use custom rom like Lineageos without Google Play services & Speech Servi
 1) Install [eSpeak TTS engine](https://f-droid.org/en/packages/com.reecedunn.espeak) available on F-Droid
 
 2) Set it as the preferred engine
-a) Go to Lineageos main **Settings**
-b) Scroll down to **Accessibility**
-c) Tap on **text-to-speech output** (below SCREEN READERS)
-d) tap on **Preferred engine** (left side) and make sure **eSpeak** is selected
+
+      Go to Lineage os main **Settings** --> Scroll down to **Accessibility** --> Select **text-to-speech output** (below SCREEN READERS)--> **Preferred engine** (left side) and make sure **eSpeak** is selected
```

Comment:
This is a markdown, you can make a numbered list.

Reply:
Is this enough?

Reply:
It's `LineageOS`, see https://lineageos.org/ or https://github.com/lineageos